resources/views/PageDesign/Partials/Header/Navigation.twig

```diff
@@ -12,7 +12,13 @@
                         <img alt="{{ companyName }}" src="{{ companyLogo }}" class="img-fluid" />
                     </a>
                 {% endif %} 
-                <button v-open-mobile-navigation class="navbar-toggler d-lg-none p-3" type="button">
+                <button v-open-mobile-navigation
+                        class="navbar-toggler d-lg-none p-3"
+                        type="button"
+                        id="mobile-navigation-toggler"
+                        for="mobile-navigation-wrapper"
```

Comment:
```suggestion
```

I don't think for attribute is allowed on anything besides <label> and <output>
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for